app/src/main/res/values-ar/strings.xml

```diff
@@ -15,13 +15,17 @@
 
     <!-- Buttons -->
     <string name="button_continue">استمرار</string>
+    <string name="button_advanced_options_show">اظهار الاعدادات المتقدمة</string>
+    <string name="button_advanced_options_hide">اخفاء الاعدادات المتقدمة</string>
+    <string name="button_select_backup_import">حدد ملف النسخة الاحتياطية للستيراد</string>
 
     <!-- Menus -->
     <string name="app_info">معلومات التطبيق</string>
     <string name="stop_app">ايقاف التطبيق</string>
-    <string name="stop_session">قتل الخدمة</string>
+    <string name="stop_session">قتل الخدمة</string>               <!-- We know this says name="kill_service" but it is really stopping a session. -->
```

Comment:
Our latest English string actually uses `"Stop Session"` now something like `توقف الجلسة` would be better (I used google translate for that 😅).  

Also you can remove the comment `<!-- We know this says name="kill_service" but it is really stopping a session. -->` since it doesn't use "kill service" anymore

Reply:
modified